### ISSUE TITLE
Pro Offline Maps: Some labels missing larger accessibility text sizes

### ISSUE THREAD

**Issue by cachly** [cachly/bug-tracker#4841]
**[Original report](https://bitbucket.org/zedsaidstudio/cachly/issues/4840) by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------
